### DISCUSSION TITLE
version shows “unknown”

### DISCUSSION THREAD

**Discussion by fate1ess** [microsoft/winget-cli#1516] [Q&A]
Why software "f.lux", the version number always shows "unknown", unable to upgrade.

![pic_9](https://user-images.githubusercontent.com/25172023/134999110-a2a0c164-9de1-4c42-9573-28d59ca52e0a.png)

**Comment by Masamune3210** [comment]
Unfortunately a thing we have to deal with for now until more heuristics to fix it are dealt with, hope I'm wrong though
